### DISCUSSION TITLE
Add Friends Via Generated Revolt QR Code

### DISCUSSION THREAD

**Discussion by UnixPhonez** [stoatchat/discussions#334] [Feature Suggestions]
The ability to add friends through a generated qr code would be pretty cool, similar to how snapchat functions

**Comment by 4444dogs** [comment]
That would be amazing because this would fix the issue of trying to find your friend's account when there aren't discriminators.

**Reply by amytimed** [reply]
wdym

**Reply by 4444dogs** [reply]
like my name is 444dogs instead of 4444dogs#5130

**Reply by ShadowLp174** [reply]
hi there :) @Alex-Sour

**Reply by ShadowLp174** [reply]
but this isn't a problem as your username exists only once @4444dogs

**Reply by 4444dogs** [reply]
> but this isn't a problem as your username exists only once @4444dogs

Yeah I guess that is true lol. But still, it would be cool.

**Reply by ShadowLp174** [reply]
Agreed :)
